webpack/ForemanInventoryUpload/ForemanInventoryConstants.js

```diff
@@ -2,4 +2,4 @@ import { translate as __ } from 'foremanReact/common/I18n';
 import { foremanUrl } from '../ForemanRhCloudHelpers';
 
 export const INVENTORY_PAGE_TITLE = __('Red Hat Inventory Uploads');
-export const INVENTORY_BASE_URL = foremanUrl('/foreman_inventory_upload/');
+export const inventoryBaseUrl = path => foremanUrl('/foreman_inventory_upload/' + path);
```

Comment:
I agree it can be a function, though in the contants file i prefer to keep it simple as:
```js
export const INVENTORY_BASE_URL = () => foremanUrl('/foreman_inventory_upload/');￼
```

Reply:
or it can be moved altogether to the helpers file